desktop-exporter/app/routes/main-view.jsx

```diff
@@ -1,20 +1,31 @@
 import React from 'react';
-import { Link } from "react-router-dom";
+import { Link, useLoaderData } from "react-router-dom";
+
+export async function loader() {
```

Comment:
```suggestion
export async function mainLoader() {
```

If you change the name here, the import in `main.jsx` will look like:

```js
import MainView, { mainLoader } from './routes/main-view';
```

which feels a little more sensible to me.

Reply:
or you could rename everything else to be consistent with the `summaryLoader` and it could be:

```js
import SummaryView, { summaryLoader } from './routes/summary-view';
```